platform/ios/src/MGLMapboxEvents.m

```diff
@@ -180,20 +135,16 @@ + (BOOL)isEnabled {
             [[NSUserDefaults standardUserDefaults] integerForKey:@"MGLMapboxAccountType"] == 0);
 }
 
++ (BOOL)debugLoggingEnabled {
```

Comment:
Yeah, this should go away. At one point I had some debug logging in MGLMapView, but those keys didn't make the final cut.